KubeNameResolver: Add support for image based gadgets (#4169)
inspektor-gadget/inspektor-gadget: docs/spec/operators/kubenameresolver.md

---
title: KubeNameResolver
---

> The KubeNameResolver enriches the `K8s.PodIp`,`K8s.HostIp` and `K8s.PodLabels` fields by reading and resolving the `K8s.PodName` and `K8s.Namespace` fields.

Reviewer: I feel "enriching a field" is wrong, it should be "enriching an event"

What about?

```suggestion
The KubeNameResolver operator uses the `K8s.PodName` and `K8s.Namespace` fields to enrich the event with `K8s.PodIp`,`K8s.HostIp` and `K8s.PodLabels` fields. 
```

Or something along the line.
